docs/source/models/supported_models.rst

```diff
@@ -484,9 +484,15 @@ Multimodal Embedding
     - Example HF Models
     - :ref:`LoRA <lora>`
     - :ref:`PP <distributed_serving>`
+  * - :code:`LlavaNextForConditionalGeneration`
+    - LLaVA-NeXT-based
+    - T / I
+    - :code:`royokong/e5-v`
+    - 
+    - ✅︎
   * - :code:`Phi3VForCausalLM`
     - Phi-3-Vision-based
-    - T + I
+    - T / I / T + I
```

Comment:
I would rather avoid that, since it would get pretty messy for models like llava-onevision. I'll add a note to clarify that `+` includes any combination of the included modalities then.